.sass-lint.yml

```diff
@@ -1,13 +1,14 @@
 files:
   include: 'src/**/*.scss'
+
 rules:
-  property-sort-order: 0
-  no-color-literals: 0
-  no-ids: 0
-  hex-length: 0
-  hex-notation: 0
   declarations-before-nesting: 0
   force-element-nesting: 0
-  shorthand-values: 0
-  no-vendor-prefixes: 0
+  hex-length: 0
+  hex-notation: 0
+  no-color-literals: 0
+  no-ids: 0
   no-qualifying-elements: 0
+  no-vendor-prefixes: 0
+  property-sort-order: 0
+  shorthand-values: 0
```

Comment:
IMO, always better to split these kinds of reorgs / sorts into separate commits, because they can obscure actual changes in diffs